README.md

```diff
@@ -16,3 +16,99 @@ IDURAR is Open Source ERP / CRM (Invoice / Inventory / Accounting / HR) Based on
 **Live App Demo** : [https://idurarapp.com/open-source-erp-crm/](https://idurarapp.com/open-source-erp-crm/) "Please use chrome"
 
 `username : admin@demo.com password : admin123`
+
+## Getting started
+
+### Step 1: Clone the repository
+
+```bash
+git clone https://github.com/idurar/idurar-erp-crm.git
+cd idurar-erp-crm
+```
+
+### Step 2: Create Your MongoDB Account and Database/Cluster
+
+- Create your own MongoDB account by visiting the MongoDB website and signing up for a new account.
+
+- Create a new database or cluster by following the instructions provided in the MongoDB documentation. Remember to note down the "Connect to your application URI" for the database, as you will need it later. Also, make sure to change `<password>` with your own password
+
+- add your current IP address to the MongoDB database's IP whitelist to allow connections (this is needed whenever your ip changes)
+
+### Step 3: Rename the Environment File
+
+Locate the file named tmp.variables.env in the root directory of the project and rename it to .variables.env. This file will be used to store the necessary environment variables for the project to run.
+
+### Step 4: Update MongoDB URI
+
+In the .variables.env file, find the line that reads:
+
+`DATABASE="your-mongodb-uri"`
+
+Replace "your-mongodb-uri" with the actual URI of your MongoDB database.
+
+### Step 5: Install Backend Dependencies
+
+In your terminal, navigate to the root directory of the project and run the following command to install the backend dependencies:
+
+```bash
+npm install
+```
+
+This command will install all the required packages specified in the package.json file.
+
+### Step 6: Run Setup Script
+
+While still in the root directory of the project, execute the following command to run the setup script:
+
+```bash
+node setup/setup.js
+```
+
+This setup script may perform necessary database migrations or any other initialization tasks required for the project.
+
+### Step 7: Run the Backend Server
+
+In the same terminal, run the following command to start the backend server:
+
+```bash
+npm run dev
+```
+
+This command will start the backend server, and it will listen for incoming requests.
+
+### Step 8: Install Frontend Dependencies
+
+Open a new terminal window , and run the following command to install the frontend dependencies:
+
+```bash
+cd frontend
+npm install
+```
+
+This command will navigate to the frontend directory within the project and install all the required packages for the frontend.
+
+### Step 9: Run the Frontend Server
+
+After installing the frontend dependencies, run the following command in the same terminal to start the frontend server:
+
+```bash
+npm run start
+```
+
+This command will start the frontend server, and you'll be able to access the website on localhost:3000 in your web browser.
+
+` If you encounter an OpenSSL error while running the frontend server, follow these additional steps:`
+
+```bash
+export NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
I am not able to understand this line of code. Can You explain?

Reply:
@rahullo this is an issue not everyone is facing, from my understanding this has to do primarily with old dependencies but which one I am not sure, this issue will be resolved when we update this repo hopefully ,
if you close terminal and want to run the project again you will have to set it again so it does not persist , this  is a temporary fix, if you can find a better solution feel free to contribute 🙏